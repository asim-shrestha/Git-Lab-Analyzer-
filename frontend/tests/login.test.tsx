import {createShallow} from "@material-ui/core/test-utils";
import React from 'react';
import Index from '../pages/login/index';

describe("login", () =>{
    // @ts-ignore
    let shallow;

    beforeAll(() => {
        shallow = createShallow();
    })

    it("Render Login", () => {
        // @ts-ignore
        const wrapper = shallow(<Index />)
        expect(wrapper).toMatchSnapshot();

    })

})