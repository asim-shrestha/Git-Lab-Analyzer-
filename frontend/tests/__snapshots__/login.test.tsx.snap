// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login Render Login 1`] = `
<Styled(MuiBox)
  alignItems="center"
  className="makeStyles-backgroundGradient-1"
  display="flex"
  height="100vh"
  justifyContent="center"
  width="100vw"
>
  <Styled(MuiBox)
    alignItems="center"
    borderRadius={16}
    boxShadow={20}
    className="makeStyles-card-2"
    display="flex"
    flexDirection="column"
    height="60vh"
    justifyContent="center"
    minHeight="400px"
    minWidth="250px"
    padding="25px"
    width="20vw"
  >
    <Image
      alt="The Gitlab Logo"
      height={100}
      src="/gitlab.svg"
      width={100}
    />
    <WithStyles(ForwardRef(Typography))
      align="center"
      variant="h6"
    >
      GitLab
      <br />
      Analyzer
    </WithStyles(ForwardRef(Typography))>
    <WithStyles(ForwardRef(TextField))
      fullWidth={true}
      placeholder="Username"
      variant="filled"
    />
    <WithStyles(ForwardRef(TextField))
      fullWidth={true}
      placeholder="Password"
      variant="filled"
    />
    <WithStyles(ForwardRef(TextField))
      fullWidth={true}
      placeholder="GitLab Token"
      variant="filled"
    />
    <WithStyles(ForwardRef(Button))
      color="primary"
      disableElevation={true}
      variant="contained"
    >
      Login
    </WithStyles(ForwardRef(Button))>
  </Styled(MuiBox)>
</Styled(MuiBox)>
`;
